crates/subspace-farmer/src/farm.rs

```diff
@@ -194,11 +194,11 @@ pub trait PieceCache: Send + Sync + fmt::Debug {
 /// Result of piece storing check
 #[derive(Debug, Copy, Clone, Encode, Decode)]
 pub enum MaybePieceStoredResult {
-    /// Definitely not stored
+    /// Piece is not stored, or it can't be added to the cache
     No,
-    /// Maybe has vacant slot to store
+    /// Cache might have a vacant slot to store this piece
     Vacant,
```

Comment:
might have vacant slot since another piece might have been stored simultaneously ?

Reply:
Or another sector might use the space the piece was going to be written to